packages/core/core/src/RequestTracker.js

```diff
@@ -462,11 +472,15 @@ export default class RequestTracker {
 
   async runRequest<TInput, TResult>(
     request: Request<TInput, TResult>,
+    force?: boolean,
```

Comment:
This should probably be an options object in case we want to add more options in the future.